force-app/main/default/triggers/AccountTriggers.trigger

```diff
@@ -0,0 +1,57 @@
+trigger AccountTriggers on Account (before insert, before update, after insert, after update) {
+
+    switch on Trigger.OperationType {
+        when  BEFORE_INSERT {
+            for(Account tempAcc: Trigger.new){
+                if(tempAcc.Type == null){
+                    tempAcc.Type = 'Prospect';
+                    tempAcc.Rating = 'Hot';                    
```

Comment:
According to the instructions, there are different parameters for updating these fields.
Try to check to see if the fields phone, website and fax are not null before updating the accounts to 'HOT'